utils/constants.ts

```diff
@@ -4,6 +4,8 @@ export const ethAddress = getAddress(`0x${'e'.repeat(40)}`);
 export const ipfsDomain = 'https://gateway.ipfs.io/ipfs/';
 export const VALID_AMOUNT_REGEX = /^[\d\\.]+$/;
 
+export const TOKEN_PRIORITY_SORT = ['USDC', 'DAI', 'USDT'];
```

Comment:
This may be a deficiency in our token list, but every chain looks good except BNB for me.

![Screenshot from 2023-04-06 20-40-10](https://user-images.githubusercontent.com/12705146/230517804-3856be44-573c-46a2-89f7-e232c3d10f45.png)

Reply:
thanks -- indeed, our token list does not include USDC, DAI, or USDT for bsc